frontend/src/utils/generalUtils.ts

```diff
@@ -91,7 +93,7 @@ export const validateSubmitTxInputs = (
   settings: ISettingsState, sourceChain: ChainEnum, destinationChain: ChainEnum,
```

Comment:
create a new validateSubmitTxInputs func for skyline where:
 if (!isNativeToken && BigInt(amount) < BigInt(settings.minUtxoValue[sourceChain])) - Amount less than minimum
if (isNativeToken && BigInt(amount) < BigInt(settings.minUtxoValue[destinationChain])) - Amount less than minimum

both chains are cardano for skyline, so you can remove nexus logic

Reply:
or you can do:
const chain = isNativeToken ? destinationChain : sourceChain
if (BigInt(amount) < settings.minUtxoValue[chain]) - Amount less than minimum